content/posts/2018-03-18-dependency-upgrade.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "The Art of a Dependency Upgrade"
```

Comment:
This is quite a provoking title… Maybe something more relaxed/cozy?

It sounds like a blog post from Robert Martin or something

Reply:
Also this paragraph below:

>I’ve pointed out a couple of patterns which might help with major dependency upgrades.

sounds like title misalignment to me, maybe "The Art of Upgrading Project Dependency" will be less confusing and more specific

Reply:
This is mostly a reference to The Art of War book and I kind of like it 😉Let’s just roll with it and face the backlash on the streets.

Reply:
Which reminds me of [a good song](https://www.youtube.com/watch?v=wMykYSQaG_c).